hip_data_tools/google/sheets/sheets.py

```diff
@@ -39,3 +46,29 @@ def get_value_matrix(self, workbook_name, sheet_name, row_range='', skip_top_row
             row = worksheet.row_values(i)
             list_of_lists.append(row)
         return list_of_lists
+
+    def get_fields_names(self, workbook_name, sheet_name):
+        """
+        Get the field names as a list
+        Args:
+            workbook_name (string): name of the workbook (eg: Tradie Acquisition Targets)
```

Comment:
what about conflicts with multiple workbooks of the same name, which one will be fetched?

Reply:
Maybe we need to consider a naming convention for the shared workbooks to avoid unexpected behaviour

Reply:
Added more details in the comment